src/lib/utils.js

```diff
@@ -20,3 +39,24 @@ export async function pollUntil( fn, interval, isDone ) {
 		await new Promise( res => setTimeout( res, interval ) );
 	}
 }
+
+/**
+ * Create a temporary directory in the system's temp directory
+ *
+ * @param {string} prefix Prefix for the directory name
+ * @return {string}				Path to the temporary directory
+ * @throws {Error} 				If the directory cannot be created
+ */
+export function makeTempDir( prefix = 'vip-cli' ) {
+	const tempDir = fs.mkdtempSync( path.join( os.tmpdir(), `${ prefix }-` ) );
+	debug( `Created a directory to hold temporary files: ${ tempDir }` );
+
+	process.on( 'exit', () => {
+		if ( fs.existsSync( tempDir ) ) {
+			fs.rmSync( tempDir, { recursive: true } );
```

Comment:
```suggestion
			fs.rmSync( tempDir, { force: true, recursive: true } );
```
No need for `fs.existsSync()` if we add the force option
https://nodejs.org/api/fs.html#fsrmsyncpath-options